src/languages/taggerscript.js

```diff
@@ -9,33 +9,36 @@ export default function(hljs) {
     className: 'comment',
     begin: /\$noop\(/,
     end: /\)/,
-    contains: [ {
+    contains: [{
+      begin: '/\\./'
```

Comment:
Is this meant to be different than ESCAPE_SEQUENCE?

Reply:
(other than the lack of highlighting)

Reply:
Not really, but my main purpose here was to catch escaped parantheses. But having this a string clearly was a bug, this is of course supposed to be a regex. So I pushed an update and changed it to `/\\[()]/`  and also extended the test case to check for this.

This is supposed to handle things like:

```
$noop(\))
$noop(\()
```

Or more complext stuff with nesting:

```
$noop(\($noop(abc$func(\))\)xyz))
```

All of the above is just a single $noop, but the () inside need to be nested properly and eventually get escaped with a backslash.